packages/dashboard/src/utils/index.js

```diff
@@ -23,3 +23,4 @@ export {
 } from './usePagePreviewSize';
 export { default as useStoryView } from './useStoryView';
 export { default as useTemplateView } from './useTemplateView';
+export { default as deepMerge } from './deepMerge';
```

Comment:
Maybe something for design system since it's used by story editor as well?

Reply:
Ah okay, I thought you prefer to duplicate it until we decide something in https://github.com/google/web-stories-wp/issues/9639. Will move to design system

Reply:
#9639 is for utils shared by the `wp-*` packages, no? Here we are in the core dashboard, not `wp-dashboard`

Reply:
Yes, correct.

Reply:
Moved in https://github.com/google/web-stories-wp/pull/9644/commits/8eb755628f1ca37180294374a63c9732ce6b1b5e